requirements_dev.in

```diff
@@ -1,4 +1,6 @@
-black
+# Lock to a fixed version of black, as upgrades change behavior which requires a change in code style
+# we want to have more control over when upgrades happen
+black==21.4b1
```

Comment:
I'm actually not locking python-black deliberately so that the code base style evolves in parallel with the python black versions. This avoids the "big bang" event where we blat the git history when we do eventually decide to upgrade python-black and run it.

Reply:
fixed